dbcon/joblist/distributedenginecomm.cpp

```diff
@@ -1163,7 +1164,30 @@ int DistributedEngineComm::writeToClient(size_t aPMIndex, const SBS& bs, uint32_
       map_tok->second->queue.push(sbs);
     }
 
+    int tries = 0;
+    // Try to setup connection with PS, it could be a situation that PS is starting.
+    // MCOL-5263.
+    while (tries < 10 && Setup())
```

Comment:
I am ok with fixing the attempts number however the sleep timeout value should be exponential b/c it is painful to wait for 5 seconds if it was a network glitch.

Reply:
Sure, will fix it.